js/data/newTabData.js

```diff
@@ -63,6 +63,6 @@ module.exports.topSites = [
     'favicon': `${iconPath}/playstore.png`,
     'location': 'https://play.google.com/store/apps/details?id=com.brave.browser',
     'themeColor': 'rgb(241, 241, 241)',
-    'title': 'Brave Browser: Fast AdBlock – Apps para Android no Google Play'
+    'title': 'Brave Browser: Fast AdBlock – Apps for Android not Google Play'
```

Comment:
If I go to the link above it says `Brave Browser: Fast AdBlocker - Apps on Google Play`

Reply:
@Jacalz

Reply:
Fixed now with latest commit :+1: